lib/actions/attachmentToJson.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const { promisify } = require('util');
+const request = promisify(require('request'));
+
+const path = require('path');
+const xml2Json = require('../xml2Json.js');
+const fs = require('fs');
+
+
+function streamToString(stream) {
+    const chunks = [];
+    return new Promise((resolve, reject) => {
+        stream.on('data', chunks.push);
+        stream.on('error', reject);
+        stream.on('end', () => resolve(Buffer.concat(chunks).toString('utf8')));
+    });
+}
+
+module.exports.process = async function processAction(msg, cfg) {
+    let xml = {};
+
+    console.log('attachment to string started, message=%j cfg=%j', msg, cfg);
+
+    const attachments = msg.attachments;
```

Comment:
Is this supposed to be the `msg.body.attachments` or the `attachments` object that lives in the header?